geowave-analytics/src/main/java/mil/nga/giat/geowave/analytics/mapreduce/kde/compare/ComparisonAccumuloStatsReducer.java

```diff
@@ -109,7 +109,7 @@ protected void reduce(
 
 	private Point2d[] fromIndexToLL_UR(
 			final long index ) {
-		final double llLon = ((Math.floor(index / numYPosts) * 360.0) / numXPosts) - 180.0;
+		final double llLon = ((Math.floor(index / (double) numYPosts) * 360.0) / numXPosts) - 180.0;
```

Comment:
But, but, floor rounds to -infinity,  int/int rounds to zero!  What if index or numYPosts is negative! :)